topostats/grains.py

```diff
@@ -183,6 +187,38 @@ def remove_small_objects(self, image: np.array, **kwargs):
             return small_objects_removed > 0.0
         return image
 
+    def area_thresholding(self, image: np.ndarray, upper: float, lower: float):
```

Comment:
I know you're already working on this, but just adding a comment for transparency. 

Perhaps we should remove the Scikit area thresholding method and replace it with Max's method using only the lower size limit. This would make the code's behaviour more consistent between configurations. (Not having two different methods of thresholding size).

Reply:
Done